src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -193,6 +197,7 @@ class WiFiManager
     static void ConnectHandler(uint8_t * data);
     static void DisconnectHandler(uint8_t * data);
     static void PostConnectivityStatusChange(ConnectivityChange changeType);
+    static void _SendRouterSolicitation(System::Layer * layer, void * param);
```

Comment:
nit: why using the underscore prefix?

Reply:
It was moved from the ConnectivityManagerImplWiFi that for some reason uses such convention for private methods. I removed the underscore.